libensemble/gen_classes/aposmm.py

```diff
@@ -43,9 +43,10 @@ def __init__(
                 ("local_pt", bool),
             ]
             gen_specs["persis_in"] = ["x", "f", "local_pt", "sim_id", "sim_ended", "x_on_cube", "local_min"]
-        super().__init__(variables, objectives, History, persis_info, gen_specs, libE_info, **kwargs)
+        super().__init__(vocs, History, persis_info, gen_specs, libE_info, **kwargs)
+
         if not self.persis_info.get("nworkers"):
-            self.persis_info["nworkers"] = kwargs.get("nworkers", gen_specs["user"]["max_active_runs"])
+            self.persis_info["nworkers"] = kwargs.get("nworkers", gen_specs["user"].get("max_active_runs", 4))
```

Comment:
I don't think we want to cap max_active_runs at 4 if it's not given. (Or at least that needs to be documented.)

Reply:
isn't there a default in aposmm? Or should be required.

Reply:
Hang on its not capping max_active_runs its setting nworkers in persis_info. It seems this may be irrelevant in an ask/tell format.